db/migrate/20200524074928_create_service_resource_attachments.rb

```diff
@@ -0,0 +1,12 @@
+class CreateServiceResourceAttachments < ActiveRecord::Migration[5.2]
+  def change
+    create_table :service_resource_attachments do |t|
```

Comment:
I'm concerned the name here will be confused with service resources (which is an actual thing as services table entries have resources).  I think `storage_service_resource_attachments` would be clearer.

Reply:
@Fryguy 
Renaming the table to storage_service_resource_attachments causes the index-name to be too long.
```ArgumentError: Index name 'index_storage_service_resource_attachments_on_storage_service_id' on table 'storage_service_resource_attachments' is too long; the limit is 63 characters```

Maybe we could use shorthand for naming the table `strg_service_resource_attachment`, but rename the ApplicationRecord to `StorageServiceResourceAttachment` ?

Reply:
You can just override the default index name with something shorter, no need to rename the table

Reply:
If the table-name is so long that auto generated indexes fail, won't it be the case with every index that involves this table?
In the future if anyone wants to add another index on this table, they'll unexpectedly get an error like I have and they'll have to specify an explicit index name.
Also, if any other table should want to reference this one, the index-name would also be unexpectedly too long. 

So my idea was that shortening the table name would make indexes involving this table "just work" and spare future developers from these surprising errors. I thought this sort of consideration was the reason for having ext_management_system in stead of external_management_system.